Gemfile

```diff
@@ -59,6 +59,8 @@ group :development do
 
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem "spring"
+
+  gem "sqlite3", "~> 1.6", ">= 1.6.6"
```

Comment:
shouldn't it be in the `:development, :test` group? I can test it tomorrow, but just at the first glance

Reply:
:+1: done in d415d45.

I also added the `timeout` config to `config/database.yml` for `development` in 0fbc943 because it's present in `default` [on a fresh app](https://github.com/rails/rails/blob/v7.1.3.3/railties/lib/rails/generators/rails/app/templates/config/databases/sqlite3.yml.tt#L10)

Reply:
Thank you!
One last question - I've missed it yesterday. Why is sqlite in version < 2 ?
In a new rails app it would be 2.1 https://github.com/rails/rails/blob/main/Gemfile#L161

do you think we can bump it to `gem "sqlite3", ">= 2.1"` as well?

Reply:
I should have explained this right away.

The upgrade to sqlite3 in version >= 2 is coming from https://github.com/rails/rails/commit/2976d3767e6572ee1838b50b5f401983918f99c7 for Rails v8.0.0.beta1.

Since this repo is using Rails 7.1.3.3, I used this as a reference (https://github.com/rails/rails/blob/v7.1.3.3/Gemfile#L157), which is `gem "sqlite3", "~> 1.6", ">= 1.6.6"`.

Reply:
thank you very much for the explanation